### PR TITLE
Launch desktop from cmd

### PR DESCRIPTION
**Pull request by maxcapodi78**
Sometimes the GRPC API fails to connect to a new AEDT Session. This new method launches AEDT in a separate thread and then tries to connect to it on a given port.

### CODE REVIEW

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -882,6 +884,16 @@ def _init_cpython_new(self, non_graphical, new_aedt_session, version, student_ve
...
                 return
+        elif new_aedt_session:
+            installer = os.path.join(base_path, "ansysedt")
+            if not is_linux:
+                installer = os.path.join(base_path, "ansysedt.exe")
+            out, self.port = self._launch_aedt(installer, non_graphical, self.port)
+            if out:
+                ScriptEnv._doInitialize(version, None, False, non_graphical, self.machine, self.port)
+            else:
+                self.logger.error("Failed to start AEDT on port ", self.port)
```
Comment (MaxJPRey) [inline pyaedt/desktop.py:895]:
```suggestion
                self.logger.error("Failed to start AEDT on port %s.", self.port)
```

File: pyaedt/desktop.py [hunk 2]
```diff
@@ -1524,3 +1536,34 @@ def install(package_path, package_name=None):
         if aedt_version >= "2023.2":
             write_toolkit_config(os.path.join(toolkit_dir, product), lib_dir, toolkit_name, toolkit=toolkit)
         self.logger.info("{} toolkit installed.".format(toolkit_name))
+
+    @pyaedt_function_handler()
+    def _launch_aedt(self, full_path, non_graphical, port, first_run=True):
+        """Launch AEDT in GRPC mode."""
```
Comment (MaxJPRey) [inline pyaedt/desktop.py:1542]:
```suggestion
        """Launch AEDT in gRPC mode."""
```

File: pyaedt/desktop.py [hunk 3]
```diff
@@ -1524,3 +1536,34 @@ def install(package_path, package_name=None):
...
+                my_env[env] = val
+            if is_linux:
+                subprocess.Popen(command, env=my_env, stdout=subprocess.DEVNULL, stderr=subprocess.PIPE)
+            else:
+                subprocess.Popen(" ".join(command), env=my_env, stdout=subprocess.DEVNULL, stderr=subprocess.PIPE)
+
+        self._aedt_process_thread = threading.Thread(target=launch_desktop_on_port)
+        self._aedt_process_thread.daemon = True
+        self._aedt_process_thread.start()
+        timeout = 180
```
Comment (MaxJPRey) [inline pyaedt/desktop.py:1559]: Maybe 180 s is too long for the timeout. What do you think?

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/desktop.py (hunk 3 above)
Comment (maxcapodi78) [reply pyaedt/desktop.py:1559]: in windows typically is true. On linux it depends if there are deps check

Comment (maxcapodi78) [reply pyaedt/desktop.py]: moved to settings.desktop_launch_timeout variable

**Review by Alberto-DM — APPROVED** [review]
LGTM

**Pull request merged**